prepare_oim/roles/deploy_containers/kubespray/tasks/deploy_kubespray_container.yml

```diff
@@ -51,21 +51,24 @@
 
 - name: Deploy Kubespray container and check deployment status
   block:
-    - name: Deploy the Kubespray container
-      containers.podman.podman_container:
-        name: "{{ kubespray_container_name }}"
-        image: "{{ kubespray_image_name }}:{{ kubespray_image_tag }}"
-        state: present
-        restart_policy: "no"
-        hostname: "{{ kubespray_container_name }}"
-        network_mode: host
-        volumes: "{{ kubespray_volumes }}"
-        cap_add:
-          - AUDIT_WRITE
+    - name: Create Quadlet file for Kubespray
+      ansible.builtin.template:
+        src: "kubespray.j2"
+        dest: "/etc/containers/systemd/{{ kubespray_container_name }}.container"
+        owner: root
+        group: root
+        mode: "0644"
```

Comment:
mode should be in vars

Reply:
@abhishek-sa1 Will be added in the next commit for sure